src/main/java/net/pms/dlna/RootFolder.java

```diff
@@ -95,7 +95,12 @@ public boolean isValid() {
 		return true;
 	}
 
+ 	@Override
 	public void discoverChildren() {
+		discoverChildren(true);
```

Comment:
@SubJunk this was not in your merging and that is why all virtual folders were missing.